packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -193,17 +195,17 @@ export class FakeChannelProvider implements ChannelProviderInterface {
     const allocations = params.allocations;
     const appData = params.appData;
 
-    log.debug(`Player ${this.getPlayerIndex()} updating channel ${channelId}`);
+    log.debug(`Player ${this.getPlayerIndex(channelId)} updating channel ${channelId}`);
     const latestState = this.findChannel(channelId);
 
     const nextState = {...latestState, participants, allocations, appData};
-    if (nextState !== latestState) {
```

Comment:
This logic seems incompatible with how we are implementing a unidirectional payment channel: namely, an implementation where the beneficiary countersigns payments. In that scenario, we often pass precisely the same state to this function, which must increment the `turnNum` and store it.

Reply:
Makes sense.